Examples/UIExplorer/js/NativeAnimationsExample.js

```diff
@@ -189,22 +244,29 @@ class EventExample extends React.Component {
             styles.block,
             {
               opacity,
-            }
+            },
           ]}
         />
         <Animated.ScrollView
           horizontal
-          style={{ height: 100, marginTop: 16 }}
+          style={{height: 100, marginTop: 16}}
           scrollEventThrottle={16}
-          onScroll={
-            Animated.event([{
-              nativeEvent: { contentOffset: { x: this.state.scrollX } }
-            }], {
+          onScroll={Animated.event(
+            [
```

Comment:
Yea :S Managed to make it format it better by putting each arg on a different line.